docs/apm/apm-alerts.asciidoc

```diff
@@ -115,8 +115,8 @@ Select **Save**. The alert has been created and is now active!
 [[apm-alert-manage]]
 === Manage alerts and actions
 
-From the APM app, select **Alerts** > **View active alerts** to be taken to the Kibana alerts and actions management page.
-From this page, you can create, edit, disable, mute, and delete alerts, and create, edit, and disable connectors.
+From the APM app, select **Alerts** > **View active alerts** to be taken to the Kibana **Rules and Connectors** page.
```

Comment:
Tested this against a 7.14 snapshot. Looks like the flow is select **Alerts and rules** > **Manage rules** to be taken to the Kibana **Rules and Connectors** page.

Reply:
How in the heck did I mess that up?

Reply:
```suggestion
From the APM app, select **Alerts and rules** > **Manage rules** to be taken to the Kibana **Rules and Connectors** page.
```